src/wrapper/data_chunk.ml

```diff
@@ -29,6 +29,20 @@ let get_opt t type_ idx =
 
 let free t ~here = Resource.free t.data_chunk ~here
 
+let to_string_hum ?(bars = `Unicode) ~column_count t =
+  match t.length, column_count with
+  | 0, _ -> "" (* Empty data chunk *)
+  | _, 0 -> "" (* No columns to display *)
+  | _, _ ->
+    let columns =
+      List.init column_count ~f:(fun idx ->
+        let name = sprintf "Column %d" idx in
+        Ascii_table_kernel.Column.create name (fun i ->
+          if i < t.length then "..." else ""))
```

Comment:
Please actually pretty-print the content of the chunk instead of eliding output, and ensure that the test outputs look reasonable.